test/cucumber-test-json/cucumber_report.json

```diff
@@ -0,0 +1,384 @@
+[
```

Comment:
I think that this file should be not uploaded, should be generated into the ci process

Reply:
Can you explain to me how to do this?
I placed the script for the json file to be created but it's still failing